build.gradle

```diff
@@ -71,3 +72,7 @@ subprojects {
 
     apply from: '../gradle/publish.gradle'
 }
+
+configure(subprojects.findAll {it.name != 'jaeger-generated'}) {
+   apply plugin: 'net.ltgt.errorprone'
```

Comment:
I looked at that earlier, and while it offers more control over where in the lifecycle this is executed, it doesn't seem to support exclusions.
Exclusions are thorny because errorprone is a javac compiler enhancement and is run in place of the plain old java compiler. By splitting `java-generated` off into it's own subproject, we can simply compile it with the regular javac and pass down the compiled files as dependencies. 
Unless there is some way of informing errorprone enhanced javac that some files are generated and it should exclude them, I don't see how this can be solved elegantly.